### PR TITLE
Create a end-to-end kubeflow example using seq2seq model (1/n)

### PR DESCRIPTION
**Pull request by ankushagarwal**
The goal is to have an end-to-end example of training and serving a non-trivial model using kubeflow.

This PR is mostly based upon kubeflow/README.md with changes tailored to this example.

Will incrementally check-in the other parts.

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/20)

### CODE REVIEW

**Comment by ankushagarwal** [comment]
@jlewi Can you review?

File: issue_summarization_github_issues/setup_a_kubeflow_cluster.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+# Setup Kubeflow
+
+In this part, you will setup kubeflow on an existing kubernetes cluster.
+
+## Requirements
```
Comment (jlewi) [inline issue_summarization_github_issues/setup_a_kubeflow_cluster.md]: Can we just link to the Kubeflow user guide as opposed to duplicating setup instructions here?

File: issue_summarization_github_issues/setup_a_kubeflow_cluster.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+```
+
+If using GKE, we can configure our cloud environment to use GCP features with a
+single parameter:
+
+```
+ks param set kubeflow-core cloud gke --env=${KSONNET_ENV}
+```
+
+### Provision storage for training data
```
Comment (jlewi) [inline issue_summarization_github_issues/setup_a_kubeflow_cluster.md]: See kubeflow/kubeflow#240 and kubeflow/kubeflow#241. Can we create docs in a central location that cover topics like how to setup storage?

@nkashy1 Were you going to take a look at doing that?

**Comment by ankushagarwal** [comment]
I've updated the PR so that it references kubeflow user guide

**Pull request merged**